### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by Corbinj22**
## Description
Added testing coverage of the UserStats component. 

Fixes # (issue)

## Type of change
Testing

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
Unit tests checking for getbyText

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/encounterState/reducer.js [hunk 1]
```diff
@@ -21,6 +23,19 @@ export const reducer = (state, action) => {
...
+        npc: {
+          ...state.npc,
+          health: state.npc.health - action.hp,
+        },
+      };
+    case EncounterActions.WIN:
+      return {
+        ...state,
+        npc: null,
+      };
```
Comment (Maxwell-Baird) [inline src/encounterState/reducer.js:38]: Do we have a defeat case yet?

File: src/App/App.js [hunk 2]
```diff
@@ -2,34 +2,77 @@ import React from "react";
 import "./App.css";
 import UserStats from "../UserStats/UserStats";
 import Map from "../Map/Map";
-import Login from '../Login/Login'
+import Login from "../Login/Login";
 import Actions from "../Actions/Actions";
 import Description from "../Description/Description";
 import Encounters from "../Encounters/Encounters";
-import { Switch, Route } from "react-router-dom";
+import FightActions from "../Fight/FightActions/FightActions";
```
Comment (rubyr) [inline src/App/App.js:9]: useless import?

File: src/Encounters/Encounters.js [hunk 3]
```diff
@@ -1,12 +1,15 @@
...
 import "./Encounters.css";
 import { useEncounterDispatch, useEncounterState } from "../encounterState";
 
 const Encounters = ({ location }) => {
   const { npc, isLoading } = useEncounterState();
   const { getNewEncounter } = useEncounterDispatch();
 
-  if (npc === null && !isLoading) getNewEncounter(location);
+  useEffect(() => {
+    debugger;
```
Comment (rubyr) [inline src/Encounters/Encounters.js:10]: remove debugger statement

File: src/Encounters/Encounters.test.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        <PlayerContextProvider>
+          <EncounterContextProvider>
+            <Encounters location="forest" />
+          </EncounterContextProvider>
+        </PlayerContextProvider>
+      </MemoryRouter>
+    );
+  });
+
+  // it("should show the user some possible actions`", () => {
```
Comment (rubyr) [inline src/Encounters/Encounters.test.js:22]: remove commented code

**Pull request merged**